frontend-react/src/components/FileHandlers/FileHandlerMessaging.test.tsx

```diff
@@ -63,6 +64,7 @@ describe("FileSuccessDisplay", () => {
 
 describe("FileErrorDisplay", () => {
     test("renders expected content", async () => {
+        const restore = conditionallySupressConsole("failure:");
```

Comment:
Gonna add this to the upcoming tickets so we don't `console.error` on all validator errors 😬